src/kixi/hecuba/onyx/jobs/measurements.clj

```diff
@@ -0,0 +1,71 @@
+(ns kixi.hecuba.onyx.jobs.measurements
+  (:require [clojure.core.async :refer [chan >! <! close! timeout go-loop]]
+            [cheshire.core :as json]
+            [taoensso.timbre :as timbre]
```

Comment:
timbre is cool but would be good to find out if it can spit out logstash format (I've only worked out with log4j until now).